src/main/java/toughasnails/season/ChunkData.java

```diff
@@ -0,0 +1,72 @@
+package toughasnails.season;
+
+import net.minecraft.world.chunk.Chunk;
+
+public class ChunkData {
+	private final ChunkKey key;
+	private Chunk chunk;
+	
+	private long lastPatchedTime;
+	private boolean bIsVisited;
```

Comment:
Guessing you come from a C++ background (or something else?) :P Usually we just stick with isVisited, isActive, toBePatched etc.